_posts/2021-04-15-jesse-frost.md

```diff
@@ -0,0 +1,81 @@
+---
+title: "Jesse Posner on Flexible Round-Optimized Schnorr Threshold Signatures (FROST)"
+permalink: /blog/2021/04/15/frost/
+layout: post
+author: jesse
+category: "technical"
+description: "Jesse Posner explains the FROST signature scheme."
+---
+
+The promise of Bitcoin is only as secure as the cryptographic keys that
+authenticate its operations: they are the keys to freedom in the digital age.
+However, the requirements for Bitcoin key management are becoming more demanding
+with innovative off-chain protocols such as the [Lightning Network][ln] and
+[Discreet Log Contracts][dlc] that introduce new roles and responsibilities for
+keys. In addition, Bitcoin's relentless rise in value creates a need for
+sophisticated systems that can help make rubber-hose cryptanalysis uneconomical.
```

Comment:
I'm not sure how widely understood "rubber-hose cryptanalysis" will be to a general audience. Perhaps use more explicit language, or link to https://en.wikipedia.org/wiki/Rubber-hose_cryptanalysis?